spec/integration/transaction-manager.spec.js

```diff
@@ -0,0 +1,98 @@
+import _ from 'lodash'
+import { expect } from 'chai'
+
+import * as utils from '../utils/helper'
+import * as constants from '../../lib/constants'
+import TransactionManager from '../../lib/services/transaction-manager'
+
+describe('TransactionManager', () => {
```

Comment:
Also here for all below tests i would go with unit instead of integration tests.

Reply:
As in my previous comment https://github.com/commercetools/commercetools-node-variant-reassignment/pull/9#discussion_r154577140 I would like to leave the tests there and we can delete them once we have some other tests which cover these use cases.